yui/apps/weather/weather.py

```diff
@@ -0,0 +1,427 @@
+    )
+
+
+def clothes_by_temperature(temperature: float) -> str:
+    if temperature <= 5:
+        return "패딩, 두꺼운 코트, 목도리, 기모제품"
+    elif temperature <= 9:
+        return "코트, 가죽재킷, 니트, 스카프, 두꺼운 바지"
+    elif temperature <= 11:
+        return "재킷, 트랜치코트, 니트, 면바지, 청바지, 검은색 스타킹"
+    elif temperature <= 16:
+        return "얇은 재킷, 가디건, 간절기 야상, 맨투맨, 니트, 살구색 스타킹"
+    elif temperature <= 19:
+        return "얇은 니트, 얇은 재킷, 가디건, 맨투맨, 면바지, 청바지"
+    elif temperature <= 22:
+        return "긴팔티, 얇은 가디건, 면바지, 청바지"
+    elif temperature <= 26:
+        return "반팔티, 얇은 셔츠, 반바지, 면바지"
+    else:
+        return "민소매티, 반바지, 반팔티, 치마"
+
+
+def get_aqi_description(aqi_level: int) -> str:
+    if aqi_level >= 5:
+        return "매우 나쁨(환자군 및 민감군에게 노출시 심각한 영향 유발, " "일반인도 유해한 영향이 유발될 수 있는 수준)"
+    elif aqi_level == 4:
+        return (
+            "나쁨(환자군 및 민감군[어린이, 노약자 등]에게 유해한 영향 유발, "
+            "일반인도 건강상 불쾌감을 경험할 수 있는 수준)"
+        )
+    elif aqi_level == 3:
+        return "민감군 영향(환자군 및 민감군에게 유해한 영향이 유발될 수 있는 수준)"
+    elif aqi_level == 2:
+        return "보통(환자군에게 만성 노출시 경미한 영향이 유발될 수 있는 수준)"
+    else:
+        return "좋음(대기오염 관련 질환자군에서도 영향이 유발되지 않을 수준)"
+
+
+@box.command("날씨", ["aws", "weather", "aqi", "공기", "먼지", "미세먼지"])
+@argument("address", nargs=-1, concat=True)
+async def weather(
+    bot,
+    event: Message,
+    address: str,
+):
+    """
+    지역의 현재 기상상태를 조회합니다.
+
+    `{PREFIX}날씨 부천` (부천으로 검색되는 주소의 현재 기상상태를 출력)
+    """
+
+    if len(address) < 2:
+        await bot.say(
+            event.channel,
+            "검색어가 너무 짧아요! 2글자 이상의 검색어를 사용해주세요!",
+        )
+        return
+
+    addr = md5(address.encode()).hexdigest()
+
+    full_address_key = f"WEATHER_ADDRESS_{addr}_full_address"
+    lat_key = f"WEATHER_ADDRESS_{addr}_lat"
+    lng_key = f"WEATHER_ADDRESS_{addr}_lng"
+    cached = await bot.cache.get_many(
+        full_address_key,
+        lat_key,
+        lng_key,
+    )
+
+    full_address = cached.get(full_address_key)
+    lat = cached.get(lat_key)
+    lng = cached.get(lng_key)
+
+    if not all([full_address, lat, lng]):
+        try:
+            full_address, lat, lng = await get_geometric_info_by_address(
+                address,
+                bot.config.GOOGLE_API_KEY,
+            )
+
+            await asyncio.gather(
+                bot.cache.set(full_address_key, full_address),
+                bot.cache.set(lat_key, lat),
+                bot.cache.set(lng_key, lng),
+            )
+        except IndexError:
+            await bot.say(event.channel, "해당 주소는 찾을 수 없어요!")
+            return
+
+    try:
+        result: tuple[
+            WeatherRecord, AirPollutionRecord
+        ] = await asyncio.gather(
+            get_weather_by_coordinate(
+                lat, lng, bot.config.OPENWEATHER_API_KEY
+            ),
+            get_air_pollution_by_coordinate(
+                lat, lng, bot.config.OPENWEATHER_API_KEY
+            ),
+        )
+
+        (weather_result, air_pollution_result) = result
```

Comment:
```suggestion
        weather_result, air_pollution_result = result
```

괄호 없어도 unpack 될 것 같습니다

Reply:
괄호가 없긴 한데, 저는 명시적으로 하는걸 좋아하는 편인데... 이런건 제 생각엔 아예 pre-commit에서 스타일을 강제해도 좋을지도요!